workers/generic-worker/runtime/runtime_windows.go

```diff
@@ -83,12 +85,12 @@ func DeleteUser(username string) (err error) {
 
 func ListUserAccounts() (usernames []string, err error) {
 	var out string
-	out, err = host.CombinedOutput("wmic", "useraccount", "get", "name")
+	out, err = host.CombinedOutput("powershell", "-Command", "Get-LocalUser | Select-Object -ExpandProperty Name")
 	if err != nil {
 		return
 	}
-	for _, line := range strings.Split(out, "\r\n") {
-		trimmedLine := strings.Trim(line, "\r\n ")
+	for _, line := range strings.Split(out, "\n") {
+		trimmedLine := strings.TrimSpace(line)
 		usernames = append(usernames, trimmedLine)
 	}
 	return
```

Comment:
Were you able to test this locally? I'm sure it will be fine, but it occurs to me that if for some reason it was broken and didn't list historical task user accounts, disks might slowly fill up over time...
Another way to check this would be to run several tasks on a worker, and then run a task to list the users or `dir` the parent of the home directory, just to make sure it really is cleaning up old users.

Reply:
Yeah, the output of `wmic` vs the powershell `Get-LocalUser` are quite different, with the powershell cmdlet outputting with just newline separators. It's a good idea though to try running multiple tasks against one worker - I'll do that now. Honestly, the `strings.TrimSpace()` isn't needed at all, too.

Reply:
Yeah, this is working as expected. https://community-tc.services.mozilla.com/provisioners/proj-taskcluster/worker-types/gw-windows-2022-staging/workers/westus/vm-cspkoykswqp4s8jzuiruqcxk4eh0pq1g1rd?sortBy=started&sortDirection=desc